src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,59 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.StringParser;
+import duke.TaskList;
+
+/**
+ * Command type find.
+ */
+public class FindCommand extends Command {
+
+    private final String command;
+
+    /**
+     * Find command builder.
```

Comment:
Shouldn't this be phrased like other methods where you start with a verb? Probably something on the lines of 'Constructs...'

Reply:
Nice suggestion, changes were made